scripts/build_wheels.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# agent0 install
+
+echo "install required packages for building wheels"
+python -m pip install --upgrade pip uv build
+uv pip install agent0@.[all]
```

Comment:
This is installing agent0 from pip, where we're then building a new version. This should be an editable install locally.
```
uv pip install -e .[all]
```

Reply:
scratch that, I think this installs a non-editable install locally, which I think is okay